tasks/section6.yml

```diff
@@ -297,16 +297,98 @@
       - notimplemented
 
 - name: "SCORED | 6.2.10 | PATCH | Ensure users' dot files are not group or world writable"
-  command: /bin/true
-  changed_when: no
+  find:
+    paths: '{{ item }}'
+    patterns: '\.[0-9A-Za-z].*'
+    file_type: file
+    follow: no
+    hidden: yes
+    recurse: yes
+    use_regex: yes
+    depth: 4
+  register: dotfiles
+  when:
+      - rhel7cis_rule_6_2_10
+  loop:
+    - /root
+    - /home
+  tags:
+      - level1
+      - level2
+      - patch
+      - rule_6.2.10
+
+- name: "SCORED | 6.2.10 | PATCH | Ensure users' dot files are not group or world writable"
+  find:
+    paths: '{{ item }}'
+    patterns: '\.[0-9A-Za-z].*'
+    file_type: directory
+    follow: no
+    hidden: yes
+    recurse: yes
+    use_regex: yes
+    depth: 4
+  register: dotdirs
+  when:
+      - rhel7cis_rule_6_2_10
+  loop:
+    - /root
+    - /home
+  tags:
+      - level1
+      - level2
+      - patch
+      - rule_6.2.10
+
+- name: "SCORED | 6.2.10 | PATCH | Ensure users' dot files are not group or world writable"
+  set_fact:
```

Comment:
I would prefer not to use set_fact like this. It can have some unintended consequences and it could just as easily be accomplished by doing `with_items: "{{dotfiles.results|list}}"` on the `file` module.

Reply:
the reason this was done is that results is an array of /root and /home.  the guy that wrote the code wasn't sure how to Nest this type of with_items situation.  perhaps i need to break out into separate executions for each path